src/azul/service/manifest_service.py

```diff
@@ -70,15 +74,30 @@
 logger = logging.getLogger(__name__)
 
 
+class ManifestFormat(Enum):
+    COMPACT = 'compact'
```

Comment:
The precedent in Azul is to use lower-case, even for enum members. There are no true constants in Python so I see no point in pretending.